### DISCUSSION TITLE
Global asset lineage not loading on VM

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17829] [Q&A]
Is this a common issue?

If I run dagster on my vm the global asset lineage never loads and give the message "Rendering a large number of assets, please wait…". This exact same project runs instantly if launched locally.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699467285631059?thread_ts=1699467285.631059&cid=C01U954MEER).*

**Comment by hellendag** [comment]
Are there any JS errors in the browser console in the case where it never loads?
